### ISSUE TITLE
Ubuntu unity 8 UBPorts.

### ISSUE THREAD

**Issue by robertoduwe** [ubports/unity8#57]
Meu intuito é ajudar relatando problemas e sugerir melhorias.
Não domino o idioma Inglês, mas deve haver Brasileiros no grupo.
Caso não seja possível contribuir no idioma Inglês, irei utilizar o tradutor do Google.
Instalei o Unity 8 sobre o Ubuntu Mini 18.04 64 bits em uma VM do Virtualbox.
A instalação ocorreu sem problemas, mas encontrei alguns problemas e deficiências.
1. Sempre, ao ligar a VM do Unity 8, a primeira tentativa de login falha e volta para a tela de login. Na segunda tentativa funciona.
2. Quando clico no painel inicial do lançador, percebo que tem poucos apps instalados. Bem menos que na versão de smartfones. Teria como incluir pelo menos os básicos que estão na versão dos smartfones?
3. Ao clicar sobre o app "Scopes", ele abre uma pequena janela com o logo do Ubuntu e fica em loop nesta janela.
4. As pastas padrão de usuário (documentos, videos, etc não são criadas). Apenas a pasta pictures é criada, mas a tradução para meu idioma não ocorre.
Obrigado pelo belo trabalho.
Att.
Roberto Duwe

**Comment by GizmoChicken** [comment]
@robertoduwe ' s report translated into English using Google translator:

> My intention is to help report problems and suggest improvements.
I do not speak English, but there must be Brazilians in the group.
If it is not possible to contribute in the English language, I will use the Google translator.
I installed Unity 8 on Ubuntu Mini 18.04 64 bit in a Virtualbox VM.
The installation occurred without problems, but I encountered some problems and shortcomings.
>
>    1. Always, when connecting the Unity 8 VM, the first login attempt fails and returns to the login screen. The second attempt works.
>    2. When I click on the launcher's home panel, I realize that there are very few apps installed. Well less than in the version of smartphones. Would you like to include at least the basics that are in the version of smartphones?
>    3. By clicking on the "Scopes" app, it opens a small window with the Ubuntu logo and is looped in this window.
>    4. The default user folders (documents, videos, etc. are not created). Only the pictures folder is created, but the translation for my language does not occur.
>
>    Thank you for the beautiful work.
>    Att.
>    Roberto Duwe

Unity8 is still in the early development preview stage.  Most of the bugs that you mention are already known to the UBports devs.

In the case of the login issues, logind support was only recently added to Mir by the Mir development team, and a few glitches in the way Unity8 interacts with Mir’s logind support still need to be worked out.

You mentioned that there aren't many apps installed.  Well, that's because you installed onto Ubuntu "Mini" which doesn't ship with many apps.  If you had installed onto standard Ubuntu, you'd see all of the apps that ship with standard Ubuntu.  That said, not all of those apps would work.  In general, most apps that can run under Wayland currently do work, but those apps that require XWayland support currently don't work. The Mir team is still working to implement XWayland support in Mir. Once that’s done, most Xapps should work with Unity8. (But some of them, like Synaptic, have other problems that currently prevent them form working on Wayland, whether with Unity8, GNOME Shell, KDE Plasma, or otherwise.)

Because the issues that you report are known, may I close your report?
